.github/workflows/vercel-deploy.yml

```
@@ -0,0 +1,38 @@
+name: Deploy to Vercel
+
+on:
+  push:
+    branches: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)